woohoo.rb

```diff
@@ -0,0 +1,5 @@
+[:a, :b].find { |_| true }
+[:a, :b].detect { |_| true }
+
+puts "SUP!"
+puts 'SUP!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.